### PR TITLE
Fix dimension requirements for transportation

### PR DESCRIPTION
**Pull request by daniel-thom**
I removed these lines because they are redundant with the multi-dimensional requirement section.

I merged #20 into this so that I could get proper test coverage.

@ahcyip This PR now takes over #20. We now have a new rule for multi_dimensional requirements as shown below - look at `__all__`.

### CODE REVIEW

File: datasets/modeled/transport/dimensions/counties.csv [hunk 1]
```diff
@@ -84,6 +84,17 @@ id,name,state
...
+02275,NA,NA
+02195,NA,NA
+02013,NA,NA
+02282,NA,NA
+02100,NA,NA
+02016,NA,NA
+02290,NA,NA
+02158,NA,NA
+02230,NA,NA
+02105,NA,NA
```
Comment (elainethale) [inline datasets/modeled/transport/dimensions/counties.csv:97]: Why are these all NAs? Presumably these are Alaskan counties that do have names?

File: datasets/modeled/transport/dimensions/counties.csv [hunk 2]
```diff
@@ -536,6 +547,7 @@ id,name,state
 13321,Worth,GA
 15001,NA,NA
 15003,NA,NA
+15005,NA,NA
 15007,NA,NA
 15009,NA,NA
```
Comment (elainethale) [inline datasets/modeled/transport/dimensions/counties.csv:552]: Same here for Hawaii

**Review by elainethale — APPROVED** [review]
I'd prefer the Alaska and Hawaii counties to have names, but am not going to hold anything up over it.

File: datasets/modeled/transport/dimensions/counties.csv (hunk 1 above)
Comment (daniel-thom) [reply datasets/modeled/transport/dimensions/counties.csv:97]: I interpreted this as Arthur saying, “the dataset has these but the project won’t use them and so we don’t care.” If we will ever use this dataset apart from the project (or as its own project), I agree that it should have names. @ahcyip Is it trivial to add these (to your branch)?

**Comment by ahcyip** [comment]
Can add for future. Open new issue for it?

**Comment by elainethale** [comment]
Thanks for adding the county names, @daniel-thom!

**Pull request merged**